server/apps/archive/archive_crop.py

```diff
@@ -124,22 +127,30 @@ def _add_crop(self, original, crop_name, doc):
         """
         renditions = original.get('renditions')
         crop_data = UploadService().get_cropping_data(doc)
-        original_image = original.get('renditions').get('original')
-        original_file = superdesk.app.media.get(original_image.get('media'), 'upload')
-        if not original_file:
-            raise SuperdeskApiError.badRequestError('Original file couldn\'t be found')
+        original_crop = renditions.get(crop_name, {})
+        if (crop_data[0] != original_crop.get('CropLeft') or crop_data[1] != original_crop.get('CropTop') or
+                crop_data[2] != original_crop.get('CropRight') or crop_data[3] != original_crop.get('CropBottom')):
```

Comment:
It would be a bit more readable to express this condition in a more concise way (only my personal opinion):

```py
fields = ('CropLeft', 'CropTop', 'CropRight', 'CropBottom')

if any(crop_data[i] != original_crop.get(name) for i, name in enumerate(fields)):
    ...
```